### ISSUE TITLE
SimpleCalendar doesn't show all Calendars

### ISSUE THREAD

**Issue by gerhardbeck** [SimpleMobileTools/Simple-Calendar#1208]
Hi,
I use SimpleCalendar to show calendars from a nextcloud via WebDav. They are switched on as synchronized calendars, but they don't show up in the app.
The gooogle calendar app shows them correctly but not simple calendar.
Software ist 6.10.0 Pro via F-Droid

**Comment by tibbi** [comment]
can you maybe figure out which calendars arent shown? I cannot really fix it without some useful into. 6.10.3 is the latest one on Google Play, get it once it becomes available on f-droid. It probably wont affect your issue though.

**Comment by gerhardbeck** [comment]
Well, they are all from a nextcloud installation.
The yellow circled one doesn't show up at all. (original nextcloud calendar)
The red circled one didn't show up for several weeks but when I deleted the icx5-Account (ical) and created it new it and forced synchro it showed up at last.
the other ones show up without problems. (they are original nextloud calendar as well as ical)
![Screenshot_20200929-181810](https://user-images.githubusercontent.com/41069802/94585628-9099cc80-0280-11eb-81a3-6977f12a9bd2.png)

In GoogleCalendar they all show up.

**Comment by tibbi** [comment]
there must be something unique about the broken ones. Are they maybe shared with someone, not created by you, or marked Private?

**Comment by gerhardbeck** [comment]
True, the yellow one is the only one shared with another member of the cloud instance.
The red one is a public ical calendar, created by another homepage.

29.09.2020 22:57:07 Tibor Kaputa <notifications@github.com>:

**Comment by tibbi** [comment]
ok Ill keep an eye on it, not sure what is the issue now.

**Comment by gerhardbeck** [comment]
Thanks. Do you need more information on anything?
Gerhard

30.09.2020 09:54:52 Tibor Kaputa <notifications@github.com>:

**Comment by tibbi** [comment]
not really I guess, will see if I can reproduce it

**Comment by thenickname** [comment]
One thing I noticed is that the "contact birthdays" calendar from nextcloud disappeared a short while ago. It used to show up in the app but not anymore. Still works in other apps though.

Special thing about this calendar is that it is auto generated and read only I think.

**Comment by michaelof** [comment]
@gerhardbeck and @thenickname how are you synching your Nextcloud, which calDAV tool?
FYI I'm using SimpleCalendar 6.11.2 F-Droid also with Nextcloud (latest version, self-hosted). Synched via DAVx5, Version 3.3.6-ose F-Droid. Don't have the issue, all calendars shown, including contact's birthdays

**Comment by gerhardbeck** [comment]
I used DavX5 and it's not the calDav Tool as Etar and Google Calendar show everything. It must simple calendar.

**Comment by vinniec** [comment]
@tibbi I don't know if it is a good way to replicate this problem but using [simpletask](https://f-droid.org/de/packages/nl.mpcjanssen.simpletask/) program it is not possible to see its tasks in simple-calendar, but it is possible but you can see them in [Etar](https://f-droid.org/de/packages/ws.xsoh.etar/) for example.
Just yesterday I reported it on the simple task issue tracker, and someone made a [generic hypothesis](https://github.com/mpcjanssen/simpletask-android/issues/710#issuecomment-740731581) about how it works.

To replicate the problem you have to:
1) share the task of simpletask with calendars going to its settings > notifications and activating "due date reminders" and "threshold date reminders
2) add tasks by clicking on the `+` icon in the simpletask main window and adding one or both dates to the task (the clock icons at the bottom right of the task creation)
3) activate caldav synchronization in simplecalendar and select simpletask item

The events of simpletask should be seen but they do not appear, but as I said, I see them from Etar.

**Comment by tibbi** [comment]
Simple Calendar doesnt even support tasks yet, of course they are not visible...

**Comment by vinniec** [comment]
Ah, I don't know how the info is organized, so i'm sorry for the mistake :)

**Comment by thenickname** [comment]
> @gerhardbeck and @thenickname how are you synching your Nextcloud, which calDAV tool?
> FYI I'm using SimpleCalendar 6.11.2 F-Droid also with Nextcloud (latest version, self-hosted). Synched via DAVx5, Version 3.3.6-ose F-Droid. Don't have the issue, all calendars shown, including contact's birthdays

syncing via DAVx5 also. As mentioned earlier, works with other calendar apps, but not Simple Calendar anymore. Used to work in the past.

**Comment by Fanboy-Studios** [comment]
Syncing with DAVx5 doesn't work for me.

Also the .ics exported file from Simple Calendar does not import properly in my Nextcloud server.

**Comment by fabianski7** [comment]
> Simple Calendar doesnt even support tasks yet, of course they are not visible...

Is this issue enough to follow this or is it necessary to create another one?

**Comment by tibbi** [comment]
> > Simple Calendar doesnt even support tasks yet, of course they are not visible...
> 
> Is this issue enough to follow this or is it necessary to create another one?

it is already reported as one of the oldest issues

**Comment by derVodi** [comment]
It seems to be a GUI refresh problem.

**Steps To Reproduce**

1. Make sure you have at least 2 (Android-)local calendars with some appointments (I used Etar to prepare this)
2. Freshly install SimpleCalendarPro
3. Activate CalDav-Sync for the local calendars

**Expected Behaviour**

The appointments should instantly show up

**Actual Behaviour**

No appointments show up

**Workaround**

1. Create a dummy appointment in SimpleCalendarPro in one of the local calendars
2. Wait some time for the GUI to refresh
3. All appointment appear then

**Comment by betsythefc** [comment]
I came here looking for a solution to this exact problem, as a google calendar shared to me would show up in Simple Calendar, but with no events. I had to grant the Samsung calendar app access to my calendars in order to see the events. It may not be the same issue here, but in case others are experiencing similar things. I'm not sure if by re-disabling that permission for the Samsung calendar app will stop syncing the events from that shared calendar or not, though.

**Comment by tibbi** [comment]
you need some sync adapter app to take care of syncing between your calendars and the device, Simple Calendar cannot do it itself as it has no internet access. Once the events are already present on your device, we can show them too.
